_posts/2015-11-13-this-week-in-rails-new-releases-exciting-rails-5-stuff-and-more.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "This week in Rails: new releases, exciting Rails 5 stuff and more!"
+categories: news
+author: imtayadeway
+published: true
+date: 2015-11-13 18:53:09 -0800
+---
+
+#### 
+
+Hey everyone - [Tim](https://twitter.com/imtayadeway) here with another monster update with all the exciting Rails-related goings-on from this week.
+
+## Featured
+
+### [Riding Rails: Snappier Development Mode in Rails 5](http://weblog.rubyonrails.org/2015/11/11/snappier-development-mode-in-rails-5/)
+
+One of the nicest things about development mode in Rails has to be the sheer convenience of running on your latest revision on each request. This of course comes at a performance cost but is about to change: instead the filesystem will notify Rails asynchronously of any changes.
+
+### [Rails 4.2.5 and 4.1.14 are here!](https://rubygems.org/gems/rails)
+
+This week saw the release of both 4.2.5 and 4.1.14 versions of Rails. Go upgrade while they're still hot!
+
+### [This Week's Rails Contributors](http://contributors.rubyonrails.org/contributors/in-time-window/20151107-20151113)
+
+This week 33 fabulous people contributed to Rails. Check out the list of [issues](https://github.com/rails/rails/issues) if you'd like to see your name up there.
+
+## New Stuff
+
+### [New! days\_in\_year method added to Time](https://github.com/rails/rails/pull/22244)
```

Comment:
I usually manually (:disappointed:) remove these because they are unnessary, but I believe it's okay to leave them in there too

The import script is pretty rough, sorry about that :sweat_smile: